modules/statics/src/networks.ts

```diff
@@ -867,6 +867,18 @@ class BabyTestnet extends Testnet implements AccountNetwork {
   explorerUrl = 'https://testnet.babylon.explorers.guru/transaction/';
 }
 
+class Mantra extends Mainnet implements AccountNetwork {
+  name = 'Mantra';
+  family = CoinFamily.MANTRA;
+  explorerUrl = 'https://explorer.mantrachain.io/MANTRA-1/tx';
```

Comment:
might result in a bug if we dont have `/` in the end
```suggestion
  explorerUrl = 'https://explorer.mantrachain.io/MANTRA-1/tx/';
```